src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -1730,6 +2054,18 @@ static ucs_status_t uct_ib_mlx5_devx_md_open(struct ibv_device *ibv_device,
         goto err_lru_cleanup;
     }
 
+    if (md->flags & UCT_IB_MLX5_MD_FLAG_INDIRECT_XGVMI) {
+        status = uct_ib_mlx5_devx_umr_init(md);
+        if (status != UCS_OK && status != UCS_ERR_UNSUPPORTED) {
+            goto err_md_close;
+        }
+    } else {
+        md->umr_cq        = NULL;
+        md->umr_qp        = NULL;
+        md->umr_mkey_hash = NULL;
```

Comment:
should we also set the to null if `(md->flags & UCT_IB_MLX5_MD_FLAG_INDIRECT_XGVMI)`, but  `uct_ib_mlx5_devx_umr_init` returned `UCS_ERR_UNSUPPORTED`